### PR TITLE
credentials hidden

### PR DESCRIPTION
**Pull request by GadAugust**
The only change made was to hide the credentials. not been able to connect to click house.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 #Importing custom fuctions - metric query function and truncate SQLite DB function
 from extract import fetch_data
-from load import load_csv_to_postgres
+from load import load_csv_to_postgrest
```
Comment (chirozie) [inline main.py:6]: you have a typo here. the postgrest instead of postgres

File: main.py [hunk 2]
```diff
@@ -18,7 +18,7 @@ def main():
     fetch_data(client=client)
 
     # load data
-    load_csv_to_postgres('tripdata.csv', 'tripdata')
+   # load_csv_to_postgres('tripdata.csv', 'tripdata')
```
Comment (chirozie) [inline main.py:21]: you have commented out the load_csv_to_postgres function. while i understand why you did this, it was for your testing. the code to be merged to the main branch should not have this line commented out.

**Review by chirozie — CHANGES_REQUESTED** [review]
Check my comments and resolve them. then commit and push again.

File: main.py (hunk 1 above)
Comment (GadAugust) [reply main.py:6]: Thank you
resolved.

File: main.py (hunk 2 above)
Comment (GadAugust) [reply main.py:21]: Thank you resolved.

**Pull request closed without merging**